### PR TITLE
3650 add tx sender to tradelogs

### PR DESCRIPTION
**Pull request by ductm54**

### CODE REVIEW

File: tradelogs/common/types.go [hunk 1]
```diff
@@ -54,9 +54,10 @@ type TradeLog struct {
 	WalletFees     []WalletFee `json:"-"`
 	IntegrationApp string      `json:"integration_app"`
 
-	IP      string `json:"ip"`
-	Country string `json:"country"`
-	UID     string `json:"uid"`
+	IP      string           `json:"ip"`
+	Country string           `json:"country"`
+	UID     string           `json:"uid"`
+	Sender  ethereum.Address `json:"tx_sender"`
```
Comment (favadi) [inline tradelogs/common/types.go]: json symbol should match variable name.

File: tradelogs/crawler_v3.go [hunk 2]
```diff
@@ -50,6 +50,11 @@ func (crawler *Crawler) assembleTradeLogsV3(eventLogs []types.Log) ([]common.Tra
 			return result, errors.New("log item has no topic")
 		}
 
+		tradeLog.Sender, err = crawler.getTxSender(log, 10*time.Second)
```
Comment (favadi) [inline tradelogs/crawler_v3.go]: so each log will require one additional request to ethereum node, which will add significant processing time to the crawler. Could you confirm that there is nothing in our current event logs that provider tx sender?

Comment (favadi) [reply tradelogs/crawler_v3.go]: Also, 

- 10*time.Second could be a const somewhere.
- there might be multiple trades in a single tx hash, and we might unnecessary get tx sender of a transaction multiple times. Does it make sense to store the map tx_hash --> sender to reduce API calls?

File: tradelogs/common/types.go (hunk 1 above)
Comment (ductm54) [reply tradelogs/common/types.go]: fixed

File: tradelogs/crawler_v3.go (hunk 2 above)
Comment (ductm54) [reply tradelogs/crawler_v3.go]: - Our current event logs does not provide tx sender.
- change 10*time.Second to default timeout
- I think we might not need map tx_hash -> sender as most of time we will not hit cache

**Pull request merged**